ckanext/canada/schemas/presets.yaml

```diff
@@ -3089,13 +3091,18 @@ presets:
     - value: GeoTIF
       mimetype: image/tiff
       replaces: [geotif]
+    - value: GeoJSON
+      mimetype: application/geo+json
+    - value: GeoPackage
```

Comment:
I've tried to use the accepted file extensions as values in this list (e.g. `GPKG` for GeoPackage) and use the names of the formats as labels so they can be translated.